### PR TITLE
Sept 2 Synchronous Meeting minutes

### PR DESCRIPTION
**Pull request by mormj**
Signed-off-by: Josh Morman <jmorman@peratonlabs.com>

### CODE REVIEW

File: minutes/2021-09-02 Synchronous GA Meeting.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Derek Kozel
+Josh Morman
+
+
+## Agenda:
+1. Discussion of the Board and GA activities (see report)
+  - This is an open discussion. Feedback is particularly welcome!
+2. Elections of the new Board.
+  - Elections are handled by our MCs, Barry and Josh.
+  - Currently, Derek, Marc, and myself have decided to run for re-election.
```
Comment (777arc) [inline minutes/2021-09-02 Synchronous GA Meeting.md:25]: myself > Martin

Comment (mormj) [reply minutes/2021-09-02 Synchronous GA Meeting.md:25]: lol - copy and paste from Martin's email

Comment (777arc) [reply minutes/2021-09-02 Synchronous GA Meeting.md:25]: hah np

File: minutes/2021-09-02 Synchronous GA Meeting.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        - Will be announced at GRCon
+    - NUMFOCUS and other grant opportunities
+
+## Election of the Board
+
+Results: Martin Braun, Derek Kozel, and Marc Lichtman unanimously re-elected to a 2 year term
+
+## Election of Jacob Gilbert to the GA
+
+Results: 12-0 in favor of inviting to GA
```
Comment (mbr0wn) [inline minutes/2021-09-02 Synchronous GA Meeting.md:45]:
```suggestion
Results: 12-0 in favor of inviting to GA
    [Note: Due to technical issues, one of our attendees could not participate in this vote].
```

**Review by dkozel — APPROVED** [review]

**Review by mbr0wn — APPROVED** [review]

**Pull request merged**